libcst/codemod/visitors/_apply_type_annotations.py

```diff
@@ -223,33 +223,49 @@ class ApplyTypeAnnotationsVisitor(ContextAwareTransformer):
         x: int = 1
 
     If the function or attribute already has a type annotation, it will not be overwritten.
+
+    To overwrite existing annotations when applying annotations from a stub,
+    use the keyword argument ``overwrite_existing_annotations=True`` when
+    constructing the codemod or when calling ``store_stub_in_context``.
```

Comment:
@pradeep90 Can you add some explanation here and mention only the `overwrite_existing_annotations ` passed by last call take effect when multiple calls happen?